src/styles/index.scss

```diff
@@ -1,3 +1,34 @@
+@import './utils/variables';
+@import './utils/mixins';
+@import 'blocks/header';
+@import 'blocks/card';
+@import 'blocks/stars';
+
+html {
+  font-family: Roboto, Arial, sans-serif;
+}
+
 body {
   margin: 0;
+  padding: 0;
+}
+
+.catalog {
+  display: grid;
+  grid-template-columns: 200px;
```

Comment:
Consider using a more flexible unit for `grid-template-columns` instead of a fixed `200px`. This will make the layout more responsive across different screen sizes.